lib/chef_backup/helpers.rb

```diff
@@ -194,9 +201,10 @@ def reconfigure_marketplace
   end
 
   def enabled_addons
-    SERVER_ADD_ONS.select do |_name, config|
+    SERVER_ADD_ONS.select do |name, config|
       begin
-        File.directory?(File.dirname(config['config_file']))
+        File.directory?(addon_install_dir(name)) &&
+          File.directory?(File.dirname(config['config_file']))
       rescue
```

Comment:
Thanks for catching that; it was a pattern left over from some other refactoring.